infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -13,9 +14,12 @@
 
 import javax.annotation.Nonnull;
 import java.lang.invoke.MethodHandles;
+import java.net.InetSocketAddress;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
+import static com.sun.xml.internal.ws.policy.sourcemodel.wspolicy.XmlToken.Optional;
```

Comment:
this is not that Optional that you want.
your optional is - java.util.Optional

or you just forget to delete this import

Reply:
removed, added by error. thanks